src/Umbraco.Tests.Integration/Services/ContentServiceEventTests.cs

```diff
@@ -20,7 +20,7 @@ namespace Umbraco.Tests.Integration.Services
         Logger = UmbracoTestOptions.Logger.Console)]
     public class ContentServiceEventTests : UmbracoIntegrationTest
     {
-        private IContentTypeService ContentTypeService => GetRequiredService<IContentTypeService>();
+        private IContentTypeService IContentTypeService => GetRequiredService<IContentTypeService>();
```

Comment:
This naming jumps out as a bit funny to me - i.e. a property with an `I` prefix, when the other similar properties don't.  Might be better as it was before, and use the fully qualified namespaces when you need to call the static methods on `ContentTypeService`?